grpc-utils/build.gradle

```diff
@@ -1,6 +1,6 @@
 plugins {
     id "java"
-    id 'java-library'
+    id "java-library"
     id "idea"
```

Comment:
question: what is the purpose of this plugin?do we need it?

Reply:
It is to use the `api` keyword of gradle, you can see https://docs.gradle.org/current/userguide/java_library_plugin.html. Perhaps simple quote is better that double quote, I don't want a mix, either one or other. But certainly simple quote should choose instead, it is used like it in the Gradle documentation.

Reply:
yes actually my question was about the pluging "idea", about the existing code. I found it here: https://docs.gradle.org/current/userguide/idea_plugin.html 
Not sure we need it, but maybe we can check later.